DESCRIPTION

```diff
@@ -30,15 +30,16 @@ Imports:
   tidyr,
   tibble,
   shiny,
+  shinydashboard,
```

Comment:
Probably we don't need to import {shinydashboard}, do we? 
We are dependent already on way to many libraries.